cnv/cnv_install/uninstalling-container-native-virtualization.adoc

```diff
@@ -0,0 +1,18 @@
+[id="uninstalling-container-native-virtualization"]
+= Uninstalling {ProductName}
+include::modules/cnv-document-attributes.adoc[]
+:context: uninstalling-container-native-virtualization
+toc::[]
+
+You can uninstall {ProductName} by using the {product-title} 4.1
+xref:../../web-console/web-console.adoc#web-console-overview_web-console[web console].
+First, delete the custom resource you created during deployment. Then, delete
+the *KubeVirt HyperConverged Cluster Operator* catalog subscription.
```

Comment:
Ah, I didn't realize on the first review that this is conceptual not procedural info so then is fine. Not sure if first is needed but only in terms of minimalism. And that's pretty minimal :-)